README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Donated by Ranjit](https://img.shields.io/badge/donated%20by-@WhatsARanjit-fb7047.svg)](#transfer-notice)
 
+We would like to inform you that during the release of version 1.0.1, the corresponding tag v1.0.0 was not created. To address this, we are publishing a new release, v1.0.1.
```

Comment:
The statement is not correct. the v1.0.0 tag was created and then deleted again.

Reply:
updated